internal/vault/secrets/ephemeral/azure_secrets_static_credentials.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package ephemeralsecrets
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/base"
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/client"
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/errutil"
+	"github.com/hashicorp/terraform-provider-vault/internal/framework/model"
+	"github.com/hashicorp/vault/api"
+)
+
+const fieldCombinedRoleMetadata = "combined_role_metadata"
+
+var _ ephemeral.EphemeralResource = &AzureStaticCredsEphemeralSecretResource{}
+
+var NewAzureStaticCredsEphemeralSecretResource = func() ephemeral.EphemeralResource {
+	return &AzureStaticCredsEphemeralSecretResource{}
+}
+
+type AzureStaticCredsEphemeralSecretResource struct {
+	base.EphemeralResourceWithConfigure
+}
+
+type AzureStaticCredsEphemeralSecretModel struct {
+	base.BaseModelEphemeral
+
+	Backend  types.String `tfsdk:"backend"`
+	Role     types.String `tfsdk:"role"`
+	Metadata types.Map    `tfsdk:"metadata"`
+
+	ClientID             types.String `tfsdk:"client_id"`
+	ClientSecret         types.String `tfsdk:"client_secret"`
+	SecretID             types.String `tfsdk:"secret_id"`
+	Expiration           types.String `tfsdk:"expiration"`
+	CombinedRoleMetadata types.Map    `tfsdk:"combined_role_metadata"`
```

Comment:
Is this one supposed to also be documented in the "attributes reference" on the doc page?